site/content/user-guide/amqp-adapter.md

```diff
@@ -63,17 +63,17 @@ The command-line client supports the following parameters (with default values):
 * `--message.address`: The AMQP 1.0 message address (default: `telemetry/DEFAULT_TENANT/4711`)
 * `--message.payload`: The message payload body (default: `'{"temp": 5}'`)
 * `--hono.client.host`: The hostname that the AMQP adapter is running on (default: `localhost`)
-* `--hono.client.port`: The port that the adapter is listening for incoming connections (default: `5672`)
+* `--hono.client.port`: The port that the adapter is listening for incoming connections (default: `4040`)
 * `--hono.client.username`: The username to use for authenticating to the adpter (default: `sensor1@DEFAULT_TENANT`)
 * `--hono.client.password`: The password to use for authenticating to the adapter (default: `hono-secret`)
 
 To run the client using the above default values, open a terminal and execute the following:
 
-    $ /hono/cli/target$ java -jar hono-cli-0.8-SNAPSHOT-exec.jar --spring.profiles.active=amqp-adapter-cli
+    $ /hono/cli/target$ java -jar hono-cli-0.9-M1-SNAPSHOT-exec.jar --spring.profiles.active=amqp-adapter-cli
```

Comment:
what about this:
```
/hono/cli$ java -jar target/hono-cli-*-exec.jar --spring.profiles.active=amqp-adapter-cli
```
That way we wouldn't need to update the text with every new release ...